integration-tests/utils/integration-helpers.js

```diff
@@ -39,18 +39,13 @@ const deleteWorkspace = async workspaceName => {
   await ajaxPage.close()
 }
 
-const withWorkspace = test => async () => {
-  const workspaceName = await makeWorkspace()
+const withWorkspace = test => async ({ context, page }) => {
```

Comment:
We should capture and forward any additional options here, to make sure this remains fully composable.